FEATURE: Cleanup docs

DanielSiepmann opened the pull request:
We didn't cover docs for all changes and therefore update them now.

DanielSiepmann committed:
TASK: Update / cleanup documentation

Code and docs have diverged a bit. We therefore need to update them.

DanielSiepmann committed:
Merge remote-tracking branch 'origin/develop' into feature/doc-cleanup

Conflicts:
    composer.json

DanielSiepmann committed:
Merge remote-tracking branch 'origin/develop' into feature/doc-cleanup

justusmoroni commented on Documentation/source/development/contribution.rst:
```diff
@@ -0,0 +1,69 @@
+
+Then setup your system::
+
+    git clone git@github.com:DanielSiepmann/search_core.git \
```
Falsch? `DanielSiepmann/search_core`?

DanielSiepmann replied:
👍

justusmoroni commented on Documentation/source/development/contribution.rst:
```diff
@@ -0,0 +1,69 @@
+
+* Make sure to follow the documented :ref:`concepts`.
+
+.. _Github: https://github.com/DanielSiepmann/search_core
```
Auch hier. `DanielSiepmann`?

DanielSiepmann replied:
👍

justusmoroni commented on Documentation/source/development/indexer.rst:
```diff
@@ -0,0 +1,21 @@
+
+As this is the "indexer", deletion of single documents is directly processed by the connection.
+
+``setIdentifier`` is called with the identifier of the current Indexer. This might be usefull to
```
`useful` ohne 2 `f`.

DanielSiepmann replied:
👍

justusmoroni commented on Documentation/source/readme.rst:
```diff
@@ -8,24 +8,21 @@ What does it do?
+`current issues`_.
 
-We are also focusing on Code Quality and Testing through `travis ci`_, `scrutinizer`_ and `codacy`_.
+We are also focusing on Code Quality and Testing through `travis ci`_, ``phpcs``, ``phpunit`` and
```
Hier fehlt nun `phan` oder?

DanielSiepmann replied:
phan does not work completely yet. There are some issues regarding TYPO3, so therefore it's not mentioned yet, still configuration is already there to use on local machine.

DanielSiepmann committed:
TASK: Fix documentation

As we have moved the repository at github, we should update all mentions
to use new url.
Also a small typo was fixed.
